tls/s2n_security_policies.c

```diff
@@ -559,6 +559,14 @@ const struct s2n_security_policy security_policy_pq_tls_1_2_2023_04_10 = {
     .ecc_preferences = &s2n_ecc_preferences_20200310,
 };
 
+const struct s2n_security_policy security_policy_pq_tls_1_3_2023_06_01 = {
+    .minimum_protocol_version = S2N_TLS13,
```

Comment:
Is there a reason the minimum is TLSv1.3 and not TLSv1.2?

Reply:
Yes, I've received guidance from panos that PQ TLSv1.2 likely won't be standardized per [this draft RFC](https://www.ietf.org/id/draft-rsalz-tls-tls12-frozen-01.html).

Reply:
Yes, if we want to disable PQ TLS 1.2, then we should set the `.kems` parameter of `kem_preferences_pq_tls_1_3_2023_06 ` to NULL that I mentioned in my other comment. Setting this `.minimum_protocol_version` to `S2N_TLS13` will break backwards compatibility with a large amount of existing clients that are only able to negotiate up to TLSv1.2 (whether they support PQ or not). I don't think that's what was intended.